oauth2/google/simple-config/microservice.conf

```diff
@@ -43,7 +49,17 @@ server {
     proxy_set_header X-GW-Scope $oauth_token_scope;
     proxy_set_header X-GW-Client-Id $oauth_token_client_id;
     proxy_set_header X-GW-User-Id $oauth_token_user_id;
+    add_header X-Cache-Status $upstream_cache_status;
+
+    # -------------------------------------------------
+    #  Configure caching
+    # -------------------------------------------------
+    proxy_cache my_cache;
+    proxy_cache_valid any 30m;
 
     proxy_pass http://microservice;
+    # content_by_lua_block {
```

Comment:
If this endpoint is only intended for proxying the request, I would remove the commented code for nginx "OK" output.